tensorflow/python/ops/math_ops.py

```diff
@@ -3212,6 +3212,12 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+  
+  Usage Example:
+    ```python
```

Comment:
Hello, to make the doctests work, you need to remove these " ```python " and " ``` " at the end of the code.